runtime/vmchk/checkclasses.c

```diff
@@ -102,7 +102,13 @@ static BOOLEAN
 verifyJ9Class(J9JavaVM *vm, J9Class *clazz, J9Class *javaLangObjectClass)
 {
 	BOOLEAN passed = verifyJ9ClassHeader(vm, clazz, javaLangObjectClass);
-	J9ClassLoader *classLoader = clazz->classLoader;
+	J9ClassLoader *classLoader;
```

Comment:
Please initialize the variable (either to NULL, or the expected value it had before, eliminating the else below).